iroha/src/lib.rs

```diff
@@ -296,6 +304,26 @@ impl Iroha {
         );
         Ok(())
     }
+
+    /// To wait for active peers(this is provisional code)
+    async fn wait_for_active_peers(&self, sorted_peers: &[PeerId]) -> Result<(), String> {
+        log::info!("Waiting for active peers");
+        for peer in sorted_peers.iter() {
```

Comment:
Also `sorted_peers` includes an IP of this peer. It is needed to check it in the loop and not wait for itself.